assets/schema/ensemble_schema.json

```diff
@@ -1849,6 +1849,35 @@
       ]
     },
     "Switch-payload": {
+      "type": "object",
```

Comment:
hey vinoth, the schema needs to add in ensemble-web-studio ts files, check the studio repo readme

Reply:
Okay I'll update in the studio once this PR is merged